src/database/queries.ts

```diff
@@ -100,3 +103,65 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Get all transactions from the `transactions` collection
+ */
+export const getAllTransactions = async (): Promise<Transaction[]> => {
```

Comment:
getAllTransactions() and getTransaction() don't work and return empty arrays. 

code to test queries in console: 

const testQueries = async () => {
  getTransaction('8HEjHtFLvaMKsyEtLd0g').then(value => {
    console.log(value);
  });
};

testQueries();